web/src/main/java/com/bazaarvoice/emodb/web/scanner/scanstatus/ScanStatus.java

```diff
@@ -59,6 +61,8 @@ public ScanStatus(@JsonProperty ("scanId") String scanId,
         _activeScanRanges = Objects.firstNonNull(activeScanRanges, ImmutableList.<ScanRangeStatus>of());
         _completeScanRanges = Objects.firstNonNull(completeScanRanges, ImmutableList.<ScanRangeStatus>of());
         _completeTime = completeTime;
+        // Adding 1 minute buffer time to the start time to give us the compaction control time. (setting the time in the future takes care of the issue of there being any in-flight compactions)
+        _compactionControlTime = DateUtils.addMinutes(startTime, 1);
```

Comment:
Inconsistency:  The compaction control time is hard-coded to one minute after `startTime`, but in `ScanUploader` that value is actually configurable.  Maybe better to pass in either the compaction control duration or the pre-computed compaction control time value itself.